app/modules/userscript-manager/userscript-executors/old/old-script-executor.js

```diff
@@ -44,7 +44,7 @@ class OldScriptExecutor extends IScriptExecutor {
 		try {
 			this.#script = null;
 			this.#script = Function(
-				"clientUtil",
+				"_clientUtil",
```

Comment:
Why? All old userscripts I know of use `clientUtil`, not `_clientUtil`.

Reply:
it said "clientUtil not found, i changed it to this and it works. probably something like this will be fixed once the new userscirpt manager is done.

Reply:
It's your usersctipt's fault, underscore was a temporal workaround to avoid getting blocked by Krunker.

Reply:
> it said "clientUtil not found, i changed it to this and it works. probably something like this will be fixed once the new userscirpt manager is done.

The new userscript manager is already done and works (as far as I'm concerned) without problems.
There may be some problems with more complex scripts, they should however be updated to the new userscript system if possible ([See this](https://github.com/Mixaz017/idkr/blob/master/app/modules/userscript-manager/userscript-executors/1_0/1-0-script-executor.js) and [this](https://github.com/Mixaz017/idkr/blob/master/Userscripts.md)).

Reply:
okay, i will try updating my script to the new syntax accordint to the docs you send and give you guys an update if it work without ``_clientUtil``

Reply:
i was able to recreate my userscript with the new userscript syntax, although i couldn't find a way to make a settigns toggle for it that would turn it on and off, still good enough. if you guys could provide a settings toggle example in the documentation, that would be very helpful. thanks.